documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: prometheus
+  # Names the service monitor.
+  name: infinispan-monitoring
```

Comment:
This name should be unique for each Infinispan CR in the namespace and seems to be equals cluster name with postfx or just cluster name like this @Crumby test code
```
smb.withNewMetadata().withName(clusterName + "-monitor").endMetadata();
```